idunn/places/pj_poi.py

```diff
@@ -506,3 +506,15 @@ def get_order_url(self):
 
     def get_quotation_request_url(self):
         return self.get_transactional_url([TransactionalLinkType.QUOTATION_REQUEST])
+
+    def get_description(self, lang):
+        if lang != "fr" or isinstance(self.data, pj_find.Listing):
```

Comment:
Sorry, maybe a dumb question, but what does this `pj_find.Listing` mean?

Reply:
This is the root model for data coming from pagejaune's endpoint "find", which is used to search for POIs. This endpoint is mostly similar to the other endpoint "info" which gives details about a POIs, but there are some subtle differences in there response format that make it hard to build a common representation.

Here, the description is just not expected to be found in the "find" endpoint, but we wouldn't display it anyway (except for some quite rare cases in the instant answer).